src/js/navButtons.js

```diff
@@ -0,0 +1,115 @@
+/*
+
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery");
+
+    /*
+     * The back and next navigation buttons
+     */
+    fluid.defaults("gpii.firstDiscovery.navButtons", {
+        gradeNames: ["fluid.viewRelayComponent", "autoInit"],
```

Comment:
I think we should start using relay components everywhere as they are the future of the infusion and will replace standard non-relay components someday.